src/main/java/io/vertx/core/json/JsonArray.java

```diff
@@ -224,6 +225,7 @@ public JsonArray getJsonArray(int pos) {
    * @param pos  the position in the array
    * @return  the Instant, or null if a null value present
    * @throws java.lang.ClassCastException if the value cannot be converted to String
+   * @throws java.time.format.DateTimeParseException if the String value is not a legal ISODATE encoded value
```

Comment:
Maybe we should specify the expected format (Not sure everyone knows what ISODATE is).